editor/src/components/canvas/canvas-strategies/escape-hatch-strategy.spec.browser2.tsx

```diff
@@ -342,3 +344,117 @@ describe('Convert to Absolute/runEscapeHatch action', () => {
     },
   )
 })
+
+describe('Convert to Absolute', () => {
+  it('Correctly uses the captured closestOffsetParentPath to determine which elements to update', async () => {
```

Comment:
there's already a `describe('Convert to Absolute/runEscapeHatch action'` in this file, which uses the `escapeHatchTestCases`. Can't you make this test be one of those?

I'm asking because Eni and I ran into the problem that the before-after for the large blocks of code is very hard to read – it's almost like a snapshot, I don't know what are the relevant assertions. The comment  `// Converting App should not result in any changes to 'sb/scene/app/child'` is the only clue we can rely on. 
The `escapeHatchTestCases` try to capture the relevant assertions instead. (of course, there's always room for improvement)

Reply:
I've just clarified the test significantly. Hopefully that should help. I was against adding it to the existing block of tests as I feel like that actually complicates this case, but also the assertions around the frame wouldn't always work here (sometimes the frame remains the same even though the element has been incorrectly converted, as it was just given an offset of 0 in both directions). Instead the test now makes it absolutely clear which part of the code is intended to change and how.